wsi_deid/web_client/stylesheets/ItemView.styl

```diff
@@ -12,6 +12,9 @@
 .wsi-deid-replace-value
   display none
 
+.wsi-deid-auximage-container
+  display block
```

Comment:
Can we use an `inline-block` instead?  This would mean fixing the size of the map's div to the size of the loaded image.  Currently, this makes the page even longer, not utilizing modern wide screens.  Also, in my testing, the map div is sometimes wider than the window resulting in having to scroll horizontally.  Since the geojs canvas will allow zooming, leaving the images smaller shouldn't be a problem.

Reply:
This has been addressed in 7524d60 and f73524d